src/controllers/auth/forgotPassword.ts

```diff
@@ -12,11 +13,29 @@ export const postForgot = async (req: Request, res: Response): Promise<Response>
   if (!errors.isEmpty()) {
     return res.status(422).json({ errors: errors.array() })
   }
-  const email = req.body.email
-  const token = createNewToken()
-  // TODO: Save bcrypt hashedToken within database passwordResets table
-  // const payload = getPayload(token)
-  // can logger.info(token) for now, otherwise we want to email this to the user.
-  logger.info(token)
-  return res.status(501).json({ email, token })
+
+  let user: User
+
+  try {
+    const email = req.body.email
+    user = await User.findOneOrFail({ where: { email } })
```

Comment:
Can shorten this to not use a local variable `email`:
```suggestion
    user = await User.findOneOrFail({ where: { email: req.body.email } })
```

Reply:
Fixed